crates/astria-conductor/src/conductor.rs

```diff
@@ -119,11 +122,11 @@ impl Conductor {
         }
 
         if !cfg.execution_commit_level.is_firm_only() {
-            // todo!("set up stuff");
             let (shutdown_tx, shutdown_rx) = oneshot::channel();
             let (sync_done_tx, sync_done_rx) = oneshot::channel();
+
             let sequencer_reader = sequencer::Reader::new(
-                cfg.initial_sequencer_block_height,
+                init_sequencer_height,
```

Comment:
The relationship between `init_sequencer_height` and how it derives from executor is very obscure at this point. Can you add some comments explaining the relationship?